### PR TITLE
Measure execution time in native_client

### LINKED ISSUE #450: Execution time measure within native client

### PR DESCRIPTION
**Pull request by lissyx**
Fixes #450

### CODE REVIEW

File: native_client/client.cc [hunk 1]
```diff
@@ -87,10 +133,17 @@ main(int argc, char **argv)
   sox_close(input);
 
   // Pass audio to DeepSpeech
-  char* result = DsSTT(ctx, (const short*)buffer, buffer_size / 2, sampleRate);
+  struct ds_result* result = LocalDsSTT(ctx, (const short*)buffer, buffer_size / 2, sampleRate);
   if (result) {
-    printf("%s\n", result);
+    printf("%s\n", result->string);
+    if ((argc == 4) && (strncmp(argv[3], "-t", 2) == 0)) {
```
Comment (Cwiiis) [inline native_client/client.cc]: It wouldn't matter too much, but should the length parameter in the strncmp be 3 so as to include the null byte? (otherwise it would match -tanything as well as -t?)

File: native_client/deepspeech.cc [hunk 2]
```diff
@@ -198,9 +198,12 @@ DsSTT(DeepSpeechContext* aCtx, const short* aBuffer, size_t aBufferSize,
 {
   float** mfcc;
   char* string;
+
```
Comment (Cwiiis) [inline native_client/deepspeech.cc]: nit: unrelated change

File: native_client/deepspeech.cc [hunk 3]
```diff
@@ -198,9 +198,12 @@ DsSTT(DeepSpeechContext* aCtx, const short* aBuffer, size_t aBufferSize,
...
   int n_frames =
     DsGetMfccFrames(aCtx, aBuffer, aBufferSize, aSampleRate, &mfcc);
+
```
Comment (Cwiiis) [inline native_client/deepspeech.cc]: nit: unrelated change

File: native_client/deepspeech.cc [hunk 4]
```diff
@@ -198,9 +198,12 @@ DsSTT(DeepSpeechContext* aCtx, const short* aBuffer, size_t aBufferSize,
...
   string = DsInfer(aCtx, mfcc, n_frames);
+
```
Comment (Cwiiis) [inline native_client/deepspeech.cc]: nit: unrelated change

Comment (lissyx) [inline native_client/deepspeech.cc]: fixed

File: native_client/deepspeech.cc (hunk 2 above)
Comment (lissyx) [inline native_client/deepspeech.cc]: fixed

File: native_client/deepspeech.cc (hunk 3 above)
Comment (lissyx) [inline native_client/deepspeech.cc]: fixed

File: native_client/client.cc (hunk 1 above)
Comment (lissyx) [inline native_client/client.cc]: 👍

File: native_client/client.cc [hunk 5]
```diff
@@ -87,10 +133,17 @@ main(int argc, char **argv)
   sox_close(input);
 
   // Pass audio to DeepSpeech
-  char* result = DsSTT(ctx, (const short*)buffer, buffer_size / 2, sampleRate);
+  struct ds_result* result = LocalDsSTT(ctx, (const short*)buffer, buffer_size / 2, sampleRate);
   if (result) {
-    printf("%s\n", result);
+    printf("%s\n", result->string);
```
Comment (Cwiiis) [inline native_client/client.cc]: Whoops, I actually missed something here - it's possible for result->string to be NULL, so there's now a possible crash - the free(result->string) and print of the string should be in a block that checks it isn't NULL first.
